### PR TITLE
Added a 'how our money is spent page' to make donors more comfortable

### PR DESCRIPTION
**Pull request by monroemann**
Please review the code to be sure this also will work well on mobile.  

Also, feel free to read it and make sure it puts me, and the company, in a good light.  I added this because one of my law school buddies wrote, "Oh, smart, so all money donated goes into your pocket!"  But that's not true.  I make no money from this now, and I haven't used ANY of the donations for me at all.  

So I thought maybe this page would help create more transparency by showing potential donors (and volunteers) exactly what the board of directors has agreed.

Thank you for taking a look, and for your feedback.

### CODE REVIEW

**Comment by taratip** [comment]
The code looks good. It's a bit too wordy in my opinion. Maybe we could say 'when budget surpasses $5000 - $125,000, ED will be paid 40% of budget' instead of having multiple lines for each budget surpassing.

**Comment by monroemann** [comment]
That’s just legalese. If you look at it, I am not always receiving 40%. I don’t automatically get raises. For example, when one day there is finally $5,000 in the bank, I’d start getting paid $177/month. But that salary remains until then we’ve earned $10,000. Only THEN would I then receive a raise to $333/month. Does that make sense?

Perhaps there is a cleaner way to express this but I haven’t figured that out yet. I’m open to any suggestions.

**Comment by chmb** [comment]
The code looks okay. I agree with Tara on shortening the payment list. Would it be possible to compress the amount earned while the budget is under 125000 since everything below that amount is at 40%?
